demisto_sdk/commands/common/hook_validations/playbook.py

```diff
@@ -52,6 +54,8 @@ def is_valid_playbook(self, is_new_playbook: bool = True, validate_rn: bool = Tr
                 self.is_delete_context_all_in_playbook(),
                 self.are_tests_configured(),
                 self.is_script_id_valid(id_set_file),
+                self._is_id_uuid(),
+                self._is_taskid_equals_id()
```

Comment:
follow PEP https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas
```suggestion
                self._is_taskid_equals_id(),
```